tests/Controller/EnMarche/SummaryManagerControllerTest.php

```diff
@@ -598,6 +598,82 @@ public function testDeleteLanguage()
     /**
      * @depends testActionsAreSuccessfulAsAdherentWithoutSummary
      */
+    public function testAddSkillWithoutSummary()
+    {
+        $summariesCount = count($this->getSummaryRepository()->findAll());
+
+        // This adherent has no summary yet
+        $this->authenticateAsAdherent($this->client, 'gisele-berthoux@caramail.com', 'ILoveYouManu');
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/espace-adherent/mon-cv');
+
+        $this->assertCount(0, $crawler->filter('.summary-skill'));
```

Comment:
Could you please add some assertions for the CV completion here and below (with the last assertions)?